### ISSUE TITLE
SSL cert issues on website

### ISSUE THREAD

**Issue by helgelol** [zen-browser/www#212]
```
❯ curl -i https://zen-browser.app
curl: (60) SSL: no alternative certificate subject name matches target host name 'zen-browser.app'
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

Zen Browser says:
```
Did Not Connect: Potential Security Issue

Zen Browser detected a potential security threat and did not continue to zen-browser.app because this website requires a secure connection.

What can you do about it?

zen-browser.app has a security policy called HTTP Strict Transport Security (HSTS), which means that Zen Browser can only connect to it securely. You can’t add an exception to visit this site.

The issue is most likely with the website, and there is nothing you can do to resolve it. You can notify the website’s administrator about the problem.
```

I've also tried from cellular network, both through the phone itself and my mac, and still get the same errors.

**Comment by NOCanoa** [comment]
![image](https://github.com/user-attachments/assets/dfbbf97e-bbac-4f03-929c-b64606b43ff3)
dont realy understand, when i go to site i get https?

**Comment by helgelol** [comment]
TL;DR:

Apparently `zen-browser.app` is flagged in the porn filter of my ISP 😅 

It works as intended when I use Cloudflare DNS.

**Comment by NOCanoa** [comment]
Ohh that's new didn't expect that 🤣

**Comment by emilekberg** [comment]
I also get some issue, however i cannot access the website what-so-ever.
Doing it in zen browser just shows this

```
The connection has timed out

An error occurred during a connection to zen-browser.app.

    The site could be temporarily unavailable or too busy. Try again in a few moments.
    If you are unable to load any pages, check your computer’s network connection.
    If your computer or network is protected by a firewall or proxy, make sure that Zen Browser is permitted to access the web.
```

I access it to download it once, for some reason. In between of it not working as well. which is even weirder. I'm not sure if it's related to this issue or not. 

A side effect of this is that the zen mod store is not working, for instance.
